### PR TITLE
Adobe Audience Manager mobile extension style updates

### PR DESCRIPTION
**Pull request by calebk1**

### CODE REVIEW

File: using-mobile-extensions/adobe-audience-manager/README.md [hunk 1]
```diff
@@ -22,73 +13,77 @@ To get started with Audience Manager, complete these steps:
...
 ```
 
-**Important**: Audience Manager depends on the Identity extension and is automatically included in the Core pod. When manually installing the Audience Manager extension, ensure that you add the `identity-1.x.x.aar` library to your project.
+{% hint style="info" %}
+Audience Manager depends on the Identity extension and is automatically included in the Core pod. When manually installing the Audience Manager extension, ensure that you add the `identity-1.x.x.aar` library to your project.
+{% endhint %}
 {% endtab %}
 
 {% tab title="iOS" %}
 1. Add the library to your project via your `Podfile` by adding `pod 'ACPAudience'`
```
Comment (praveek) [inline using-mobile-extensions/adobe-audience-manager/README.md:35]: AEPAudience

File: using-mobile-extensions/adobe-audience-manager/audience-manager-api-reference.md [hunk 2]
```diff
@@ -141,7 +157,9 @@ This API returns the visitor profile that was most recently obtained. For easy a
 }];
 ```
 
-**Swift**
+#### Swift
```
Comment (addb) [inline using-mobile-extensions/adobe-audience-manager/audience-manager-api-reference.md:160]: it should be ### swift right?

File: using-mobile-extensions/adobe-audience-manager/audience-manager-api-reference.md [hunk 3]
```diff
@@ -110,22 +126,22 @@ Audience.getVisitorProfile(visitorProfileCallback);
...
 {% hint style="info" %}
-Method `getVisitorProfileWithCompletionHandler` was added in ACPAudience version 2.1.0.
+The `getVisitorProfileWithCompletionHandler` method was added in ACPAudience version 2.1.0.
 {% endhint %}
 
-This API returns the visitor profile that was most recently obtained. For easy access across multiple launches of your app, the visitor profile is saved in `NSUserDefaults`. If no signal has been submitted, nil is returned.
+This API returns the visitor profile that was most recently obtained. For easy access across multiple launches of your app, the visitor profile is saved in `NSUserDefaults`. If no signal has been submitted, `nil` is returned.
 
-#### **Syntax**
+#### Objective-C
```
Comment (addb) [inline using-mobile-extensions/adobe-audience-manager/audience-manager-api-reference.md:134]: it should be `### Objective-C` right?

File: using-mobile-extensions/adobe-audience-manager/audience-manager-api-reference.md [hunk 4]
```diff
@@ -159,10 +177,12 @@ ACPAudience.getVisitorProfile { (visitorProfile, error) in
 {% endtab %}
 
 {% tab title="React Native" %}
-**JavaScript**
-
 ### getVisitorProfile
 
+#### JavaScript
```
Comment (addb) [inline using-mobile-extensions/adobe-audience-manager/audience-manager-api-reference.md:182]: ###

File: using-mobile-extensions/adobe-audience-manager/audience-manager-api-reference.md [hunk 5]
```diff
@@ -171,19 +191,21 @@ ACPAudience.getVisitorProfile().then(profile => console.log("AdobeExperienceSDK:
...
 
-This API registers an extension class that was derived from `ACPExtension` with a unique name. This call validates the parameters to ensure that the name is not empty, the name is unique, and that the parent class is ACPExtension.
+This API registers an extension class that was derived from `ACPExtension` with a unique name. This call validates the parameters to ensure that the name is not empty, the name is unique, and that the parent class is `ACPExtension`.
 
 {% tabs %}
 {% tab title="Android" %}
 ### registerExtension
 
-#### **Syntax**
+#### Java
```
Comment (addb) [inline using-mobile-extensions/adobe-audience-manager/audience-manager-api-reference.md:200]: `###`

File: using-mobile-extensions/adobe-audience-manager/audience-manager-api-reference.md [hunk 6]
```diff
@@ -193,33 +215,37 @@ Audience.registerExtension();
 {% tab title="iOS" %}
 ### registerExtension
 
-#### **Syntax**
+#### Objective-C
```
Comment (addb) [inline using-mobile-extensions/adobe-audience-manager/audience-manager-api-reference.md:218]: `###`

Comment (addb) [reply using-mobile-extensions/adobe-audience-manager/audience-manager-api-reference.md:218]: I see lots of places where we need to make this change.

**Review by addb — APPROVED** [review]
Looks good, just need to fix and standardize formatting for platform name before sample code.

**Comment by emdobrin** [comment]
@calebk1 can you update based on review and update with latest from base branch?

**Comment by calebk1** [comment]
@emdobrin  - Yup, just haven't had time to work on this recently as I've been busy with other asks. I'll pick this up as soon as I can again.

**Comment by calebk1** [comment]
@emdobrin I've made the requested changes! Let me know if we're all good, and once we're approved, we should be good to merge.

**Review by emdobrin — APPROVED** [review]

**Pull request merged**